### PR TITLE
feat(home): add filters

### PR DESCRIPTION
**Pull request by andradeigor**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,7 +1,66 @@
 import CardCourse from '@/components/CardCourse';
 import data from '@/data/data.json';
+import { useEffect, useState } from 'react';
+
+interface ICourseData {
```
Comment (samueldurantes) [inline src/pages/index.tsx:5]: prefer `type` instead `interface`

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,7 +1,66 @@
...
+  campus: string;
+  course: string;
+  cutoff_mark: number;
+  group: string;
+  shift: string;
+}
 
 const Home = () => {
+  const [courses, setCourses] = useState<string[]>([]);
+  const [couresData, setCouresData] = useState<ICourseData[]>(data);
```
Comment (samueldurantes) [inline src/pages/index.tsx:16]: typo, `courses` instead `coures`

**Review by samueldurantes — APPROVED** [review]
Good job!!

I will resolve these conflicts, but before, I would like fix these comments and run `yarn prettier --write .`

**Comment by andradeigor** [comment]
done

**Pull request merged**